<html>
<head>
<title>TestNG:  2</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>2</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>4/1/1</td>
</tr><tr>
<td>Started on:</td><td>Mon Jan 29 18:56:50 IST 2018</td>
</tr>
<tr><td>Total time:</td><td>116 seconds (116787 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyRewardTabAndClickedRewardTab()'><b>verifyRewardTabAndClickedRewardTab</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense</td>
<td><div><pre>org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 30.05 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &apos;3.0.1&apos;, revision: &apos;1969d75&apos;, time: &apos;2016-10-18 09:49:13 -0700&apos;
System info: host: &apos;DESKTOP-ICBR68V&apos;, ip: &apos;192.168.1.131&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_151&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 3dc93771-cb0c-4fd7-bd44-dd15521626ed
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:274)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy7.click(Unknown Source)
	at com.test.xcdhr.Core_Sence_Check.uiActions.JoinerFirstPage.clickRewardTab(JoinerFirstPage.java:450)
	at com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyRewardTabAndClickedRewardTab(TC001_VerifyJoiner_WithLicense.java:99)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &apos;3.0.1&apos;, revision: &apos;1969d75&apos;, time: &apos;2016-10-18 09:49:13 -0700&apos;
System info: host: &apos;DESKTOP-ICBR68V&apos;, ip: &apos;192.168.1.131&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_151&apos;
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:9528)
	at &lt;anonymous class&gt;.Utils.getElementAt(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10202)
	at &lt;anonymous class&gt;.fxdriver.preconditions.visible(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10714)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12688)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12705)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:720)
... Removed 33 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace144445623", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace144445623'><pre>org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
Command duration or timeout: 30.05 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &apos;3.0.1&apos;, revision: &apos;1969d75&apos;, time: &apos;2016-10-18 09:49:13 -0700&apos;
System info: host: &apos;DESKTOP-ICBR68V&apos;, ip: &apos;192.168.1.131&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_151&apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=45.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 3dc93771-cb0c-4fd7-bd44-dd15521626ed
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:274)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy7.click(Unknown Source)
	at com.test.xcdhr.Core_Sence_Check.uiActions.JoinerFirstPage.clickRewardTab(JoinerFirstPage.java:450)
	at com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyRewardTabAndClickedRewardTab(TC001_VerifyJoiner_WithLicense.java:99)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.runSuites(TestNG.java:1028)
	at org.testng.TestNG.run(TestNG.java:996)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element not found in the cache - perhaps the page has changed since it was looked up
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &apos;3.0.1&apos;, revision: &apos;1969d75&apos;, time: &apos;2016-10-18 09:49:13 -0700&apos;
System info: host: &apos;DESKTOP-ICBR68V&apos;, ip: &apos;192.168.1.131&apos;, os.name: &apos;Windows 10&apos;, os.arch: &apos;amd64&apos;, os.version: &apos;10.0&apos;, java.version: &apos;1.8.0_151&apos;
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:9528)
	at &lt;anonymous class&gt;.Utils.getElementAt(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10202)
	at &lt;anonymous class&gt;.fxdriver.preconditions.visible(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10714)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12688)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12705)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/firoz/AppData/Local/Temp/anonymous2013785261468851180webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:720)
</pre></div></td>
<td>38</td>
<td>com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.clickOnSearchedEmployee()'><b>clickOnSearchedEmployee</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense</td>
<td></td>
<td>11</td>
<td>com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6</td></tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.searchCreatedEmployee()'><b>searchCreatedEmployee</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense<br>Parameters: payrollautoregress@xcdhr.com, london2014, Y</td>
<td></td>
<td>53</td>
<td>com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6</td></tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyEmployeeNameOnJobRoleAndGoBackToEmployment()'><b>verifyEmployeeNameOnJobRoleAndGoBackToEmployment</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense</td>
<td></td>
<td>0</td>
<td>com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6</td></tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyJobTitleOfCreatedEmployee()'><b>verifyJobTitleOfCreatedEmployee</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense</td>
<td></td>
<td>5</td>
<td>com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense.verifyAnnualSalaryOfCreatedCompensation()'><b>verifyAnnualSalaryOfCreatedCompensation</b><br>Test class: com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense</td>
<td><div><pre>java.lang.Throwable: Method TC001_VerifyJoiner_WithLicense.verifyAnnualSalaryOfCreatedCompensation()[pri:0, instance:com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6] depends on not successfully finished methods
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1340644388", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1340644388'><pre>java.lang.Throwable: Method TC001_VerifyJoiner_WithLicense.verifyAnnualSalaryOfCreatedCompensation()[pri:0, instance:com.test.xcdhr.Core_HR_Sence_Check.JoinerProcess.TC001_VerifyJoiner_WithLicense@544a2ea6] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:878)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
	at org.testng.TestNG.runSuites(TestNG.java:1028)
	at org.testng.TestNG.run(TestNG.java:996)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>null</td></tr>
</table><p>
</body>
</html>